test/browser/screenshot.js

```diff
@@ -26,6 +26,40 @@ server.listen(0, function() {
   var chrome = browser({ size: [800, 600], permissions: ['tabs'] }, function() {
     var originalExtensionSend = chrome.extension.send;
 
+    test('options are validated', function(t) {
+
+      t.test('format is validated', function(t) {
+
+        var _err = null;
+
+        try {
+          screenshot(chrome, { format: 'gif' }, function (){});
+        } catch (err) {
+          _err = err;
+        } finally {
+          t.equal(_err instanceof TypeError, true);
```

Comment:
`_err.message` is an implementation detail. If the message changes, but the error is still a `TypeError`, that is fine. This is a functional test, to ensure that the right type of error is thrown.